apps/native-component-list/src/screens/WebBrowserScreen.tsx

```diff
@@ -176,6 +184,15 @@ export default class WebBrowserScreen extends React.Component<object, State> {
   renderAndroidChoices = () =>
     Platform.OS === 'android' && (
       <>
+        <View style={styles.label}>
+          <Text>Secondary toolbar color (#rrggbb):</Text>
+          <TextInput
+            style={styles.input}
+            placeholder="RRGGBB"
+            onChangeText={this.handleSecondaryToolbarColorInputChanged}
+            value={this.state.secondaryToolbarColor}
+          />
+        </View>
```

Comment:
Shouldn't we render it only on Android? 🤔 Would be easier to test during QA 😉

Reply:
It is only on android. 'renderAndroidChoices' is used only on this one platform :)

Reply:
Ah, right, I haven't looked above 🤦‍♂️